code/_onclick/hud/human.dm

```diff
@@ -1,467 +1,475 @@
+	inv_box.icon = ui_style
+	inv_box.icon_state = "gloves"
+	inv_box.screen_loc = ui_gloves
+	inv_box.slot_id = SLOT_GLOVES
+	inv_box.hud = src
+	toggleable_inventory += inv_box
+
+	inv_box = new /obj/screen/inventory()
+	inv_box.name = "eyes"
+	inv_box.icon = ui_style
+	inv_box.icon_state = "glasses"
+	inv_box.screen_loc = ui_glasses
+	inv_box.slot_id = SLOT_GLASSES
+	inv_box.hud = src
+	toggleable_inventory += inv_box
+
+	inv_box = new /obj/screen/inventory()
+	inv_box.name = "ears"
+	inv_box.icon = ui_style
+	inv_box.icon_state = "ears"
+	inv_box.screen_loc = ui_ears
+	inv_box.slot_id = SLOT_EARS
+	inv_box.hud = src
+	toggleable_inventory += inv_box
+
+	inv_box = new /obj/screen/inventory()
+	inv_box.name = "head"
+	inv_box.icon = ui_style
+	inv_box.icon_state = "head"
+	inv_box.screen_loc = ui_head
+	inv_box.slot_id = SLOT_HEAD
+	inv_box.hud = src
+	toggleable_inventory += inv_box
+
+	inv_box = new /obj/screen/inventory()
+	inv_box.name = "shoes"
+	inv_box.icon = ui_style
+	inv_box.icon_state = "shoes"
+	inv_box.screen_loc = ui_shoes
+	inv_box.slot_id = SLOT_SHOES
+	inv_box.hud = src
+	toggleable_inventory += inv_box
+
+	inv_box = new /obj/screen/inventory()
+	inv_box.name = "belt"
+	inv_box.icon = ui_style
+	inv_box.icon_state = "belt"
+//	inv_box.icon_full = "template_small"
+	inv_box.screen_loc = ui_belt
+	inv_box.slot_id = SLOT_BELT
+	inv_box.hud = src
+	static_inventory += inv_box
+
+	throw_icon = new /obj/screen/throw_catch()
+	throw_icon.icon = ui_style
+	throw_icon.screen_loc = ui_drop_throw
+	throw_icon.hud = src
+	hotkeybuttons += throw_icon
+
+	rest_icon = new /obj/screen/rest()
+	rest_icon.icon = ui_style
+	rest_icon.screen_loc = ui_above_movement
+	rest_icon.hud = src
+	static_inventory += rest_icon
+
+	internals = new /obj/screen/internals()
+	internals.hud = src
+	infodisplay += internals
+
+	healths = new /obj/screen/healths()
+	healths.hud = src
+	infodisplay += healths
+
+	healthdoll = new /obj/screen/healthdoll()
+	healthdoll.hud = src
+	infodisplay += healthdoll
+
+	pull_icon = new /obj/screen/pull()
+	pull_icon.icon = ui_style
+	pull_icon.update_icon()
+	pull_icon.screen_loc = ui_above_intent
+	pull_icon.hud = src
+	static_inventory += pull_icon
+
+	lingchemdisplay = new /obj/screen/ling/chems()
+	lingchemdisplay.hud = src
+	infodisplay += lingchemdisplay
+
+	lingstingdisplay = new /obj/screen/ling/sting()
+	lingstingdisplay.hud = src
+	infodisplay += lingstingdisplay
+
+	devilsouldisplay = new /obj/screen/devil/soul_counter
+	devilsouldisplay.hud = src
+	infodisplay += devilsouldisplay
+
+	// austation -- Adding Bloodsuckers by Coolsurf6
+	blood_display = new /obj/screen/bloodsucker/blood_counter	// FULPSTATION: Blood Volume
+	infodisplay += blood_display
+	vamprank_display = new /obj/screen/bloodsucker/rank_counter	// FULPSTATION: Vampire Rank
+	infodisplay += vamprank_display
+	sunlight_display = new /obj/screen/bloodsucker/sunlight_counter	// FULPSTATION: Sunlight
+	infodisplay += sunlight_display
```

Comment:
there's no //austation end. also no fulpstation

Reply:
Done in next commit